pegasus/helpers/section_api_helpers.rb

```diff
@@ -107,10 +107,10 @@ def self.update_if_allowed(params, dashboard_user_id)
     user_to_update = Dashboard.db[:users].where(id: params[:id], deleted_at: nil)
     return if user_to_update.empty?
     return if Dashboard.db[:sections].
-      join(:followers, section_id: :sections__id).
-      where(sections__user_id: dashboard_user_id, sections__deleted_at: nil).
-      where(followers__student_user_id: params[:id], followers__deleted_at: nil).
-      empty?
+        join(:followers, section_id: :sections__id).
+        where(sections__user_id: dashboard_user_id, sections__deleted_at: nil).
+        where(followers__student_user_id: params[:id], followers__deleted_at: nil).
+        empty?
```

Comment:
My belief is that the linter believes in four space indents following an `if` or `unless`. Granted, I'm all for consistency also. Do we want to do two everywhere, though it makes it hard to distinguish the if condition from the block inside the if? Four for line continuation? Something else?